fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/logging/LogMessageKeys.java

```diff
@@ -232,6 +232,7 @@
 
     // query plan
     PLAN("plan"),
+    PLAN_HASH("plan_hash"),
```

Comment:
As a heads up, be on the look out for conflicts with #1484

Reply:
Ack